.gitignore

```diff
@@ -29,3 +29,6 @@ coverage/
 # Ignore benchmarks output
 perf.log
 perf.svg
+
+# PyCharm
+.idea/
```

Comment:
you can add this to your global git setting, no need to do it per-project

Reply:
I believe that having `.idea/` in `.gitignore` makes the project more contributor-friendly a bit, but I can revert these changes if you want.

Reply:
We consider it a bad practice to include IDE-specific exclusions, because there is a wide variety of IDEs people use. PyCharm might be popular, but so are VSCode, vim, and many others. It does not make sense to keep maintaining exclusion lists of all of them when each user can define their own preference in their global git setting once.